scripts/copydocs.py

```diff
@@ -231,7 +231,7 @@ def copy(src, dest, keys, comment, recursive=False, saved=None, cache=None):
         cache = {}
 
     def get_many(keys):
-        docs = marshal(src.get_many(keys).values())
+        docs = marshal(src.get_many(keys, 100).values())
```

Comment:
```suggestion
        docs = marshal(src.get_many(keys, max_length=100).values())
```
Let's include the keyword for clarity.